packages/mui-material/src/ListItem/ListItem.d.ts

```diff
@@ -35,13 +35,13 @@ export interface ListItemBaseProps {
   /**
    * The container component used when a `ListItemSecondaryAction` is the last child.
    * @default 'li'
-   * @deprecated
+   * @deprecated Use the `component` or `slots.root` prop instead. This prop will be removed in v7. See [Migrating from deprecated APIs](/material-ui/migration/migrating-from-deprecated-apis/) for more details.
```

Comment:
"Use the `component` or `slots.root` prop instead" is not semantically correct since `slots.root` is not a prop, but I think it's ok. Let me know what you think.